### PR TITLE
chore: remove eslint disable

### LINKED ISSUE #153: Cover all `*.ts` with a linter
Currently, most parts of the code base are not covered with a linter and `eslint-disable` should not be used anywhere

### PR DESCRIPTION
**Pull request by BeroBurny**
resolves #153 and there are things I missed in #166

### CODE REVIEW

**Review by enemycnt — DISMISSED** [review]
I've fixed fee oracle tests.

File: packages/sdk/src/chains/EVM/__test__/utils.test.ts [hunk 1]
```diff
@@ -24,14 +20,15 @@ describe('createProposalExecutionEventListener', () => {
...
-    expect(bridge.filters.ProposalExecution).toHaveBeenCalledWith(null, null, null);
+    expect(bridge.filters.ProposalExecution.bind(this)).toHaveBeenCalledWith(null, null, null);
 
     // Call the event handler with some dummy data to ensure that the callback is called correctly
     const originDomainId = 3;
     const depositNonce = BigNumber.from('1');
     const dataHash = '0x12345';
     const tx = {};
 
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
```
Comment (BeroBurny) [inline packages/sdk/src/chains/EVM/__test__/utils.test.ts:31]: this

File: packages/sdk/src/chains/EVM/__test__/utils.test.ts [hunk 2]
```diff
@@ -54,6 +51,7 @@ describe('createProposalExecutionEventListener', () => {
...
```
Comment (BeroBurny) [inline packages/sdk/src/chains/EVM/__test__/utils.test.ts:54]: this

**Review by BeroBurny — COMMENTED** [review]
Used `eslint-disable-next-line` (not happy for it) in few cases as had no energy to research what is going on to fix types

**Comment by enemycnt** [comment]
@BeroBurny Unfortunately, the substrate and evm tests are broken becuase of the changes. So we can continue fixing them here or rollback/transfer this changes to the next PR and merge all fixes before that.

**Comment by BeroBurny** [comment]
> @BeroBurny Unfortunately, the substrate and evm tests are broken becuase of the changes. So we can continue fixing them here or rollback/transfer this changes to the next PR and merge all commits before that.

All checks are passing?

**Comment by BeroBurny** [comment]
> > @BeroBurny Unfortunately, the substrate and evm tests are broken becuase of the changes. So we can continue fixing them here or rollback/transfer this changes to the next PR and merge all commits before that.
> 
> All checks are passing?

Oh, see now CI is not covering tests.

Fixed issues I did cause, but there are left few broken from prehistoric time
> `packages/sdk/integration/test/utils.browser.test.ts`
> `packages/sdk/integration/test/connectors.test.ts`

**Review by enemycnt — APPROVED** [review]
I've enabled unit tests in CI. Integration test will be disabled for now. It's plan to update it in this task #4 #5

**Comment by BeroBurny** [comment]
> I've enabled unit tests in CI. Integration test will be disabled for now. It's plan to update it in this task #4 #5

Did rollback changes as it is out of scope of this PR

**Review by enemycnt — APPROVED** [review]
ok

**Review by wainola — APPROVED** [review]

**Pull request merged**